internal/host/host_test.go

```diff
@@ -996,33 +998,33 @@ var _ = Describe("UpdateInventory", func() {
 				expectedInstallationDisk: ""},
 			{testName: "No previous installation disk, one disk in inventory",
 				currentInstallationDisk:  "",
-				inventoryDisks:           []*models.Disk{{Name: diskName}},
-				expectedInstallationDisk: hostutil.GetDeviceFullName(diskName)},
+				inventoryDisks:           []*models.Disk{{ID: diskId, Name: diskName}},
+				expectedInstallationDisk: diskId},
 			{testName: "No previous installation disk, two disks in inventory",
 				currentInstallationDisk:  "",
-				inventoryDisks:           []*models.Disk{{Name: diskName}, {Name: otherDiskName}},
-				expectedInstallationDisk: hostutil.GetDeviceFullName(diskName)},
+				inventoryDisks:           []*models.Disk{{ID: diskId, Name: diskName}, {Name: otherDiskName}},
+				expectedInstallationDisk: diskId},
 			{testName: "Previous installation disk is set, new inventory still contains that disk",
-				currentInstallationDisk:  hostutil.GetDeviceFullName(diskName),
-				inventoryDisks:           []*models.Disk{{Name: diskName}},
-				expectedInstallationDisk: hostutil.GetDeviceFullName(diskName)},
+				currentInstallationDisk:  diskId,
+				inventoryDisks:           []*models.Disk{{ID: diskId, Name: diskName}},
+				expectedInstallationDisk: diskId},
 			{testName: "Previous installation disk is set, new inventory still contains that disk, but there's another",
-				currentInstallationDisk:  hostutil.GetDeviceFullName(diskName),
-				inventoryDisks:           []*models.Disk{{Name: diskName}, {Name: otherDiskName}},
-				expectedInstallationDisk: hostutil.GetDeviceFullName(diskName)},
+				currentInstallationDisk:  diskId,
+				inventoryDisks:           []*models.Disk{{ID: diskId, Name: diskName}, {Name: otherDiskName}},
+				expectedInstallationDisk: diskId},
 			{testName: `Previous installation disk is set, new inventory still contains that disk, but there's another
 						disk with higher priority`,
-				currentInstallationDisk:  hostutil.GetDeviceFullName(diskName),
-				inventoryDisks:           []*models.Disk{{Name: otherDiskName}, {Name: diskName}},
-				expectedInstallationDisk: hostutil.GetDeviceFullName(diskName)},
+				currentInstallationDisk:  diskId,
+				inventoryDisks:           []*models.Disk{{Name: otherDiskName}, {ID: diskId, Name: diskName}},
```

Comment:
forgot otherDiskID here and in some other places too

Reply:
+1

Reply:
done. I didn't add it where the first disk should be selected.